doc/contribute/documentation/guidelines.rst

```diff
@@ -1006,6 +1006,13 @@ Doxygen API documentation
 
       .. doxygengroup:: can_interface
 
+
+   Doxygen project can optionally be specified using the ``:project:`` option.
+   This can be useful when multiple Doxygen projects are configured. Example::
+
+      .. doxygengroup:: can_interface
+         :project: zephyr
+
```

Comment:
do you mind using a construct similar to https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/contribute/documentation/guidelines.rst?plain=1#L1086-L1091?
Beyond the arguably more consistent formatting, this also gives us an HTML anchor "for free" that allows to directly link to the doc of that particular option should one wish to. Thanks!

Reply:
done